### PR TITLE
Add more SsmsMin interactions

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
- Added property dialog menu items for most property dialogs in SSMS
- Added GSW menu item
- Fixed bug with ExtHostObjectExplorerNode getParent function so it works correctly when objects have / in their names

### CODE REVIEW

File: extensions/admin-tool-ext-win/package.json [hunk 1]
```diff
@@ -1,61 +1,75 @@
...
+			],
+			"objectExplorer/item/context": [
+				{
+					"command": "adminToolExtWin.launchSsmsMinGswDialog",
+					"when": "isWindows && connectionProvider == MSSQL && nodeType && nodeType == Database",
+					"group": "z-AdminToolExt@1"
+				},
+				{
+					"command": "adminToolExtWin.launchSsmsMinPropertiesDialog",
+					"when": "isWindows && connectionProvider == MSSQL && nodeType && nodeType =~ /Server|Database|Table|Column|Index|Statistic|View|ServerLevelLogin|ServerLevelServerRole|ServerLevelCredential|ServerLevelServerAudit|ServerLevelServerAuditSpecification|StoredProcedure|ScalarValuedFunction|TableValuedFunction|AggregateFunction|Synonym|Assembly|UserDefinedDataType|UserDefinedType|UserDefinedTableType|Sequence|User|DatabaseRole|ApplicationRole|Schema|SecurityPolicy|SecurityPolicie|ServerLevelLinkedServer/",
```
Comment (kburtram) [inline extensions/admin-tool-ext-win/package.json]: I wonder if we should add some kind of shortcut to reference this node collection easier?  Or is this is just a one-off condition.

File: extensions/admin-tool-ext-win/src/main.ts [hunk 2]
```diff
@@ -145,10 +238,34 @@ function launchSsmsDialog(action: string, connectionProfile: azdata.IConnectionP
...
 }
+
+/**
+ * Builds the URN string for a given ObjectExplorerNode in the form understood by SsmsMin
+ * @param serverName The name of the Server to use for the Server segment
+ * @param node The node to get the URN of
+ */
+export async function buildUrn(serverName: string, node: azdata.objectexplorer.ObjectExplorerNode): Promise<string> {
+	let urnNodes: string[] = [];
+	while (node !== undefined) {
```
Comment (kburtram) [inline extensions/admin-tool-ext-win/src/main.ts]: I think `while (node) {` gets both `null` and `undefined`

File: extensions/admin-tool-ext-win/package.json (hunk 1 above)
Comment (Charles-Gagnon) [reply extensions/admin-tool-ext-win/package.json]: This should just be a one-off. It is tied to the mapping table I have in main but given that I don't see us adding many more nodes to this I'm just going to leave it as is for now and revisit later if it seems worth the effort. 

---
In reply to: [279865064](https://github.com/Microsoft/azuredatastudio/pull/5267#discussion_r279865064) [](ancestors = 279865064)

File: extensions/admin-tool-ext-win/package.json [hunk 3]
```diff
@@ -1,61 +1,75 @@
 {
-  "name": "admin-tool-ext-win",
-  "displayName": "Database Admin Tool Extensions for Windows",
```
Comment (shueybubbles) [inline extensions/admin-tool-ext-win/package.json]: should we use "Administrative" or "Administration" instead of Admin?

**Pull request merged**